src/kit.ts

```diff
@@ -455,12 +459,16 @@ const VsGenerators: {[key: string]: string} = {
 };
 
 async function varsForVSInstallation(inst: VSInstallation, arch: string): Promise<Map<string, string>|null> {
-  const common_dir = path.join(inst.installationPath, 'Common7', 'Tools');
-  const devbat = path.join(common_dir, 'VsDevCmd.bat');
-  const variables = await collectDevBatVars(devbat, ['-no_logo', `-arch=${arch}`]);
+  log.debug(`varsForVSInstallation path:'${inst.installationPath}' version:inst.installationVersion`);
+  let devbat = path.join(inst.installationPath, 'VC', 'Auxiliary', 'Build', 'vcvarsall.bat');
+  if (inst.majorVersion < 15) {
```

Comment:
`majorVersion` is not an attribute on on `VSInstallation`. Use `parseInt(inst.installationVersion)` like seen able on line 351